ros_gz_sim/launch/ros_gz_sim.launch.py

```diff
@@ -13,59 +13,100 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Launch gzsim in a component container."""
+"""Launch gzsim + ros_gz_bridge in a component container."""
 
-import launch
-from launch.actions import DeclareLaunchArgument
-from launch.substitutions import LaunchConfiguration, TextSubstitution
-from launch_ros.actions import ComposableNodeContainer
-from launch_ros.descriptions import ComposableNode
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution, TextSubstitution
+from launch_ros.substitutions import FindPackageShare
 
 
 def generate_launch_description():
 
-    world_sdf_file_arg = DeclareLaunchArgument(
+    config_file = LaunchConfiguration('config_file')
```

Comment:
Any luck with refactoring these into a separate Python module so we don't have to duplicate code?

Reply:
> Any luck with refactoring these into a separate Python module so we don't have to duplicate code?

Apparently this is possible by creating a separate launch file for the common parts. I'll do it in a separate PR.